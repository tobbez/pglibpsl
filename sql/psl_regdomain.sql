CREATE EXTENSION pglibpsl;
SELECT psl_regdomain('example.co.uk');
SELECT psl_regdomain('a.example.co.uk');
SELECT psl_regdomain('a.example.co.UK');
SELECT psl_regdomain('');
DROP EXTENSION pglibpsl;
