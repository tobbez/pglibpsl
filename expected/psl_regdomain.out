CREATE EXTENSION pglibpsl;
SELECT psl_regdomain('example.co.uk');
 psl_regdomain 
---------------
 example.co.uk
(1 row)

SELECT psl_regdomain('a.example.co.uk');
 psl_regdomain 
---------------
 example.co.uk
(1 row)

SELECT psl_regdomain('a.example.co.UK');
 psl_regdomain 
---------------
 example.co.uk
(1 row)

SELECT psl_regdomain('');
 psl_regdomain 
---------------
 
(1 row)

DROP EXTENSION pglibpsl;
